_includes/github.js

```diff
@@ -41,3 +58,13 @@ GithubRepository.prototype.startDownload = function () {
 }
 
 
+/**
+ * get href query param by key
+ */
+GithubRepository.prototype.getQueryParam = function(param) {
+  var result = window.location.search.match(
+    new RegExp("(\\?|&)" + param + "(\\[\\])?=([^&]*)")
+  );
+
+  return result ? result[3] : false;
```

Comment:
I personally like it false, since I always compare with (!!index). It makes 0 return false even when it's set.

Btw, actual false value is returned as string "false". So it never overlap.

It's up to you, in our case there wasn't a difference ( ONLY IF WE DON'T USE !! COMPARISON)
This method I have found on stack before, I liked it, was the best so far, didn't wanted to change it, after I verified that it never overlap with actual false value.

Reply:
Yuttapongaon29@gmail.com​